assets/src/components/v2/dup/dup_rotation_wrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactNode } from "react";
+
+export const splitRotationFromPropNames = (WrappedComponent: React.ElementType, rotation: string) => {
+  return class extends React.Component {
+    render() {
+
+      const modifiedProps = Object.keys(this.props).reduce((newProps: any, currentKey: string) => {
```

Comment:
We could use `_.mapKeys` to do this transformation, which would probably be more compact.

Reply:
oooo nifty indeed